jaeger_client/thrift.py

```diff
@@ -12,17 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from past.builtins import basestring
 import six
 import socket
 import struct
 
-import jaeger_client.thrift_gen.zipkincore.ZipkinCollector as zipkin_collector
+import jaeger_client.thrift_gen.jaeger.ttypes as ttypes
```

Comment:
is the plan to add a zipkinthrift converter and a zipkin http reporter? If so, a lot of this code seems pretty useful to keep track of.

Reply:
the plan is to drop zipkin format. It's not useful over UDP, but if someone wants to implement an http sender with zipkin.thrift, this code can be resurrected from git history. I don't want to waste time on keeping it since nobody asked for it.